tests/algorithms/test_connected.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from tests.conftest import edgelist1
+import xgi
+from xgi.exception import XGIError
+from xgi.readwrite import edgelist
+
+
+def test_is_connected(edgelist1, edgelist3):
+    # edgelist1 = [[1,2,3],[4],[5,6],[6,7,8]]
+    # edgelist2 = [[1,2,3],[3,4],[4,5,6]]
+    el1 = edgelist1
+    el2 = edgelist3
+    H1 = xgi.Hypergraph(el1)
+    H2 = xgi.Hypergraph(el2)
```

Comment:
I believe the first two lines are not achieving anything, no?
```suggestion
    H1 = xgi.Hypergraph(edgelist1)
    H2 = xgi.Hypergraph(edgelist2)
```

Reply:
Fixed!